pkg/daemons/config/types.go

```diff
@@ -25,6 +25,7 @@ const (
 	FlannelBackendHostGW          = "host-gw"
 	FlannelBackendIPSEC           = "ipsec"
 	FlannelBackendWireguardNative = "wireguard-native"
+	FlannelBackendTailscale       = "tailscale"
```

Comment:
Is it needed to update the `flannel-backend` flag info with the new type of backend? Is it possible to configure tailscale as backend without `vpn-auth` definition?

Reply:
If the user connects the local tailscale instance to the tailscale server (by logging in), then things will work if user passes tailscale as the flannel-backend. I am not sure if we want to document that, it will probably create confusion